citadel/hotkeys.md

```diff
@@ -34,6 +34,18 @@
 
   <tr><td>Align to world frame</td><td>Shift</td><td>hold Shift while manipulating a model to temporarily align axis its to the world frame</td></tr>
 
+  <tr><th colspan="3" class="sub"><b>MAIN WINDOW</b></th></tr>
+
+  <tr><td>Save Config</td><td>Ctrl + s</td><td>hold Ctrl and press "s" for Save the current window configuration</td></tr>
```

Comment:
A general comment for all items:

* Use "to + verb" instead of "for + verb"
* Use lowercase unless it's the name of the simulator, or of an option (like "Translate" above is the name of a mode, so it's capitalized)

That is:

```suggestion
  <tr><td>Save Config</td><td>Ctrl + s</td><td>hold Ctrl and press "s" to save the current window configuration</td></tr>
```

Reply:
Fixed in d772dad